common/men_at_arms_types/temp_draft_doc.txt

```diff
@@ -0,0 +1,587 @@
+	low_maintenance_cost = { gold = cost_55_low_maint_cost }
+	high_maintenance_cost = { gold = cost_55_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = arachnathid
+}
+jormungar_tunneler = {
+	type = siege_weapon
+	fights_in_main_phase = no
+	
+	damage = 0
+	toughness = 10
+	
+	siege_tier = 2
+	siege_value = 1.7
+	
+	buy_cost = { gold = cost_115_recruitment_cost }
+	low_maintenance_cost = { gold = cost_115_low_maint_cost }
+	high_maintenance_cost = { gold = cost_115_high_maint_cost }
+	
+	stack = 10
+	icon = jormungar_tunneler
+}
+
+jormungar_wyrm = { 
+	type = massive_infantry
+	
+	damage = 125
+	toughness = 140
+	pursuit = 5
+	screen = 35
+	
+	terrain_bonus = {
+		taiga = { damage = 20 toughness = 20 pursuit = 25 screen = 25 }
+	}
+	
+	winter_bonus = {
+		mild_winter = { damage = 5 toughness = 5 pursuit = 5 screen = 5 }
+		normal_winter = { damage = 5 toughness = 5 pursuit = 5 screen = 5 }
+		harsh_winter = { damage = 5 toughness = 5 pursuit = 5 screen = 5 }
+	}
+
+	counters = {
+		skirmishers = 1
+	}
+
+	buy_cost = { gold = cost_180_recruitment_cost }
+	low_maintenance_cost = { gold = cost_180_low_maint_cost }
+	high_maintenance_cost = { gold = cost_180_high_maint_cost }
+	
+	stack = 50
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = jormungar_wyrm
+}
+
+venomancer = {
+	type = caster
+	
+	damage = 55
+	toughness = 25
+	pursuit = 30
+	screen = 30
+	
+	terrain_bonus = {
+		taiga = { damage = 20 toughness = 20 pursuit = 25 screen = 25 }
+	}
+
+	winter_bonus = {
+		mild_winter = { damage = 5 toughness = 5 }
+		normal_winter = { damage = 5 toughness = 5 }
+		harsh_winter = { damage = 5 toughness = 5 }
+	}
+
+	counters = {
+		pikemen = 1
+		massive_infantry = 2
+		heavy_infantry = 2
+	}
+
+	buy_cost = { gold = cost_120_recruitment_cost }
+	low_maintenance_cost = { gold = cost_120_low_maint_cost }
+	high_maintenance_cost = { gold = cost_120_high_maint_cost }
+	
+	stack = 75
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = venomancer
+}
+
+webspinner = {
+	type = archers
+	
+	damage = 35
+	toughness = 20
+	pursuit = 35
+	screen = 20
+	
+	terrain_bonus = {
+		mountains = { damage = 12 toughness = 10 }
+		taiga = { damage = 12 toughness = 10 }
+	}
+
+	counters = {
+		skirmishers = 1
+		light_aerial = 1
+	}
+
+	buy_cost = { gold = cost_105_recruitment_cost }
+	low_maintenance_cost = { gold = cost_105_low_maint_cost }
+	high_maintenance_cost = { gold = cost_105_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = webspinner
+}
+#Mantid
+kunchong = { 
+	type = massive_infantry
+	
+	damage = 131
+	toughness = 125
+	pursuit = 15
+	screen = 20
+	
+	terrain_bonus = {
+		steppe = { damage = 10 toughness = 10 pursuit = 10 }
+		mountains = { damage = 10 toughness = 10 pursuit = 10 }
+		plains = { damage = 5 toughness = 5 pursuit = 5 }
+	}
+
+	counters = {
+		heavy_cavalry = 1
+		skimishers = 3
+	}
```

Comment:
+5 on all terrain bonuses (to make more difference compared to the 100+ base values) and -1 counter vs skirmishers to make up for it.